### PR TITLE
Add new constraint types to intro table

### PR DESCRIPTION
**Pull request by Hunterness**
- Added relationship key constraints as they were missing (node key existed)
- Added property uniqueness constraints cause there were none of them
- Added the new property type constraints (introduced in https://github.com/neo4j/docs-cypher/pull/597)

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-operations-834.surge.sh

**Review by l-heemann — APPROVED** [review]

File: modules/ROOT/pages/introduction.adoc [hunk 1]
```diff
@@ -230,11 +230,19 @@ a| APOC 450+ link:https://neo4j.com/docs/apoc/5/[Core Procedures and Functions]
...
+| Property existence constraints
+|
+| {check-mark}
+
+| Property type constraints
+|
 | {check-mark}
 
-| Node Key constraints
+| Node and Relationship Key constraints
```
Comment (AlexicaWright) [inline modules/ROOT/pages/introduction.adoc:245]:
```suggestion
| Node and relationship key constraints
```

**Review by AlexicaWright — COMMENTED** [review]
Avoid Camel Caps.

File: modules/ROOT/pages/introduction.adoc (hunk 1 above)
Comment (Hunterness) [reply modules/ROOT/pages/introduction.adoc:245]: I just followed what was already there, but will update :)

**Pull request merged**